ninetofiver/settings.py

```diff
@@ -134,12 +134,27 @@ def _process_cfg(cls):
     # Database
     # https://docs.djangoproject.com/en/1.10/ref/settings/#databases
 
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    if os.getenv('TRAVIS', None):
```

Comment:
Use a separate django configuration which inherits from dev/prod/whatever, instead of adding random flags. See https://github.com/kalmanolah/925r/blob/master/ninetofiver/settings.py#L279 for an example. See https://django-configurations.readthedocs.io/en/stable/ for how to select the right configuration when testing using Travis.